File: Iniciante/URI 1013.go
package main
 
import (
    "fmt"
)
 
func main() {
    var valor1, valor2, valor3 int

    fmt.Scanf("%d", &valor1)
    fmt.Scanf("%d", &valor2)
    fmt.Scanf("%d", &valor3)

	if (valor1 > valor2 && valor1 > valor3) {
		fmt.Printf("%d eh o maior\n", valor1)
	} else if (valor2 > valor3) {
		fmt.Printf("%d eh o maior\n", valor2)
	} else {
		fmt.Printf("%d eh o maior\n", valor3)
	}
}

File: Iniciante/URI 1013.rb
line = gets.split(' ')
valor1 = line[0].to_i
valor2 = line[1].to_i
valor3 = line[2].to_i

if valor1 > valor2 && valor1 > valor3
    puts "#{valor1} eh o maior"
elsif valor2 > valor3
    puts "#{valor2} eh o maior"
else
    puts "#{valor3} eh o maior"
end